Automattic/wp-calypso
# FSE:  Add back button from wp admin windowed mode

#### Changes proposed in this Pull Request

* close-button-override updated to work for windowed mode (wp-admin)
* now hides original close__toolbar and appends our custom version regardless of fullscreen mode

**Before (windowed)**
![Screen Shot 2019-10-31 at 2 35 09 PM](https://user-images.githubusercontent.com/28742426/67976099-d15e1100-fbeb-11e9-82c9-a040f5b92c73.png)
**After (windowed)**
![Screen Shot 2019-10-31 at 2 33 59 PM](https://user-images.githubusercontent.com/28742426/67976124-e20e8700-fbeb-11e9-9e9d-d83381d6af2b.png)

**Before (fullscreen)**
![Screen Shot 2019-10-31 at 2 35 36 PM](https://user-images.githubusercontent.com/28742426/67976107-d6bb5b80-fbeb-11e9-868e-be1a988dc48b.png)
**After (fullscreen)**
![Screen Shot 2019-10-31 at 2 34 18 PM](https://user-images.githubusercontent.com/28742426/67976127-e33fb400-fbeb-11e9-9d2f-6935c9490ac1.png)


#### Testing instructions

Testing with Sandbox
1.  Run this PR.
2.  navigate to edit a page through calypso interface
3.  observe new back button renders and old back arrow does not
4.  verify new back button returns to calypso page list.
5.  navigate to wp-admin
6.  verify new back button is available in windowed view and returns to wp-admin page list.

Testing with local Gutenberg environment
1. Run this PR.
2. navigate to wp-admin
3. verify new back button is available in windowed mode and returns to wp-admin page list.
4. verify new back button is available in fullscreen mode and that old back arrow is gone.




Fixes #36769, Fixes #36610, Fixes #36607
